Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -478,8 +490,8 @@ public let CodeGenerators: [CodeGenerator] = [
         }
     },
 
-    CodeGenerator("BreakGenerator", inContext: .loop) { b in
-        assert(b.context.contains(.loop))
+    CodeGenerator("BreakGenerator", inContext: ProgramContext([.loop, .switchCase])) { b in
```

Comment:
I think this is incorrect, because it'll be interpreted as "must be in the .loop **and** the .switchCase" context (while we'd want "or"): https://github.com/googleprojectzero/fuzzilli/blob/10d5db4700b2d02481b916302269025cee9a1f69/Sources/Fuzzilli/Core/ProgramBuilder.swift#L834

I think the easiest way to "fix" this for now is just to duplicate this code generator and call one of them `LoopBreakGenerator` and the other `SwitchCaseBreakGenerator` maybe? Otherwise, it'd probably need a couple more internal changes...

Reply:
Sounds good! I'll add a comment so that we can look into improving this at a later stage